pages/ugm2024.md

```diff
@@ -16,7 +16,23 @@ The UGM will bring together users of the MDAnalysis package from different commu
 * Hackathon and other social outings
 
 ## Keynote Speakers
-Keynote speakers will be announced in Spring 2024.
+### Dr. Antonia Mey
+
+<img
+src="/public/images/UGM2024photo_Mey.jpg"
+title="Dr. Antonia Mey" alt="Dr. Antonia Mey"
+style="float: left; width: 220px; height: 220px; border-radius: 20px; border: 15px solid white" />
+
+Antonia has worked on Markov State models with Frank Noe and alchemical free energy calculations with Julien Michel before starting her own research group in 2020. Her group works on computational biophysics problems using machine learning and simulations to understand protein function. She is an advocate for open-source software and diamond open access publishing models. She has contributed to pyEmma, BioSimSpace and she and her group (https://mey-research.org) are avid users of MDAnalysis. 
```

Comment:
I'd add where she has her position

... her own research group in 2020 at the [University of Edinburgh in the School of Chemistry](http://www.chem.ed.ac.uk/)

Reply:
maybe not needed to link... I just copied and pasted

Reply:
Good suggestion, thanks! I've decided not to link as we haven't linked in other places.